app/component/summary/summary-row.scss

```diff
@@ -1,5 +1,6 @@
 .itinerary-summary-row {
   padding-left: 0.5em;
+  padding-right: 0.5em;
```

Comment:
.padding-small is likely good for normal uses, but I think in this case, where there is a definite lack of space, we probably just want to optimize it to the max by itself. E.g. using .padding-small here will add more extra vertical whitespace around this component.

Anyway, I added an overflow hidden here, because the last leg text was causing horizontal scrollbars to appear in some cases.